parser.go

```diff
@@ -202,6 +202,8 @@ func (p *parser) readValue(in []byte, ignoreContinuation, ignoreInlineComment bo
 	var valQuote string
 	if len(line) > 3 && string(line[0:3]) == `"""` {
 		valQuote = `"""`
+	} else if line[0] == '"' {
+		valQuote = `"`
```

Comment:
According to https://github.com/go-ini/ini#working-with-values:

![image](https://user-images.githubusercontent.com/2946214/29246246-b1e185b2-7fc1-11e7-8f9f-74914bc35c51.png)

So this won't be supported as default behavior.